catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java

```diff
@@ -570,7 +584,7 @@ public void exitConstant_expression(BqsParser.Constant_expressionContext ctx) {
         @Override
         public void enterDate(BqsParser.DateContext ctx) {
             print(ctx.getText());
-            dateStr = "";
+            dateStr = normalizeSearchString(ctx.getText());
```

Comment:
Not sure if this is relevant, but the last errors I was seeing when querying a CSD was:
"Unable to query source, org.codice.alliance.nsili.common.UCO.InvalidParameter" - "BadQueryValue: Bad query value: The input value to compare DATETIMEMODIFIED is not a valid date". Is it possible that the normalizeSearchString is putting out an invalid date string? Again, didn't get a chance to look any further in to it - might need to accept this and then write a bug ticket later when we can actually confirm if there is actually a problem.